tests/fixtures/site_solr_disable

```diff
@@ -56,34 +54,33 @@
 -
     request:
         method: GET
-        url: 'https://onebox/api/site-names/behat-tests'
+        url: 'https://terminus.pantheon.io/api/site-names/behat-tests'
```

Comment:
Not sure if this is actually used but I assumed VCR would match on request urls looking for onebox?